versioning/BRM/extentmap.cpp

```diff
@@ -2092,8 +2106,10 @@ void ExtentMap::grabEMIndex(OPS op)
   else if (fPExtMapIndexImpl_->getShmemImplSize() != (unsigned)fEMIndexShminfo->allocdSize)
   {
     fPExtMapIndexImpl_->refreshShm();
```

Comment:
`refreshShm()` should be under WRITE lock.

Reply:
Same as previously.

Reply:
The write op in `refreshShm()` is not synchronised. Here is the race scenario:
  Reader1: Execute `ExtentMap::getExtents()` -> `fPExtMapIndexImpl_->find()`
  Reader2: Execute `ExtentMap::getExtents()` -> `grabEMEntryTable(READ)` -> `fPExtMapIndexImpl_->refreshShm()`

Reply:
After some consideration I agree with the point.